### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by cbeer**

### CODE REVIEW

**Review by mejackreed — CHANGES_REQUESTED** [review]
Nice job on this! This is working well for me. I took the Parker bibtex and annotation stuff and noticed another `try` here: https://github.com/sul-dlss/exhibits/blob/master/app/models/feature_flags.rb#L23

Also, there seem to be two more `try`s. Do we need to update those also?

Besides that everything is working well for me.

**Review by mejackreed — APPROVED** [review]

File: lib/traject/dor_config.rb [hunk 1]
```diff
@@ -243,7 +243,7 @@
 
 # PARKER FIELDS
 
-to_field 'manuscript_number_tesim', (accumulate { |resource, *_| resource.smods_rec.location.shelfLocator.try(:text) })
+to_field 'manuscript_number_tesim', (accumulate { |resource, *_| resource.smods_rec.location.shelfLocator&.text })
```
Comment (mejackreed) [inline lib/traject/dor_config.rb:246]: Should there be a `respond_to?` in here also?

**Review by jkeck — COMMENTED** [review]
Quick Q, otherwise @camillevilla and I have reviewed and looks good.

**Review by camillevilla — APPROVED** [review]

File: config/initializers/spotlight_initializer.rb [hunk 2]
```diff
@@ -31,7 +31,28 @@
...
+
+    data.merge({
+      thumbnail_square_url_ssm: riiif.image_path(resource.upload_id, region: 'square', size: '100,100'),
+      large_image_url_ssm: riiif.image_path(resource.upload_id, region: 'full', size: '!1000,1000'),
+    })
+  end)
+
+  Spotlight::Etl::Context.error_reporter = lambda do |pipeline, exception, data|
+    if pipeline.context.resource.is_a? DorHarvester
+      message = exception.inspect.truncate(1.megabyte)
```
[the remark this answers is not in the document]
Comment (cbeer) [reply config/initializers/spotlight_initializer.rb:47]: This is the first I've run into `truncate_bytes `, but, yeah, 1mb of characters was the intent. It was:

https://github.com/sul-dlss/exhibits/pull/1989/files#diff-f992e55b881f855664e3a8c00a859308530b3c11edc9bce46890f29868ea8db9L131

but 1000 character was never enough to be useful 🤷‍♂️

[the remark this answers is not in the document]
Comment (jkeck) [reply config/initializers/spotlight_initializer.rb:47]: Ah, looks like #truncate_bytes was introduced in rails 6 FWIW.  It seems like that would be a lot of data, even after truncation, but if that's what you want :shipit:

**Review by jkeck — APPROVED** [review]

**Pull request merged**